doc/howto/instances_routed_nic_vm.md

```diff
@@ -0,0 +1,42 @@
+(instances-routed-nic-vm)=
+# How to add a routed NIC device to a virtual machine
+
+When adding a [routed NIC device](../reference/devices_nic.md#nictype-routed) to an instance, the instance must be configured to use the link-local gateway IPs as default routes.
```

Comment:
```suggestion
When adding a {ref]`routed NIC device <nic-routed>` to an instance, you must configure the instance to use the link-local gateway IPs as default routes.
```

Reply:
Using MyST references has the advantage that they keep working if we move files around, or move the section into a different file.

Reply:
Done.